### PR TITLE
Build new schemes

### PR DESCRIPTION
**Pull request by jocelynthode**
Fixes https://github.com/RRethy/nvim-base16/issues/51

### CODE REVIEW

**Comment by jocelynthode** [comment]
@RRethy ready for review

File: README.md [hunk 1]
```diff
@@ -55,100 +59,196 @@ base16-black-metal-mayhem
...
+base16-windows-highcontrast-light
+base16-windows-highcontrast
+base16-windows-nt-light
+base16-windows-nt
 base16-woodland
 base16-xcode-dusk
-base16-zenburn
-```
+base16-zenburn```
+````
```
Comment (RRethy) [inline README.md:254]: The formatted for the backticks is a bit off, only use 3 at the top and bottom of the codeblock.

**Review by RRethy — COMMENTED** [review]
One nit with readme, when fixed I'll merge. Thanks a lot for contributing this.

File: README.md (hunk 1 above)
Comment (jocelynthode) [reply README.md]: Fixed, thanks for noticing the typo!

**Pull request merged**